src/sentry/api/endpoints/group_integration_details.py

```diff
@@ -138,3 +138,52 @@ def post(self, request, group, integration_id):
 
         # TODO(jess): return serialized issue
         return Response(status=201)
+
+    def delete(self, request, group, integration_id):
+        # note here externalIssue refers to `ExternalIssue.id` wheras above
+        # it refers to the id from the provider
+        external_issue_id = request.GET.get('externalIssue')
+        if not external_issue_id:
+            return Response({'detail': 'External ID required'}, status=400)
+
+        organization_id = group.project.organization_id
+        try:
+            # check org permissions
+            integration = OrganizationIntegration.objects.filter(
+                integration_id=integration_id,
+                organization_id=organization_id,
+            ).select_related('integration').get().integration
```

Comment:
why does this use filter and get?

Reply:
Also, why does this query OrganizationIntegration, when you only want the Integration? Just query for the Integration directly and avoid the extra data here.

Reply:
@MaxBittker the filter then get is so i can do `.select_related('integration')` and save the additional query.

@mattrobenolt the reason i was querying through `OrganizationIntegration` is to verify that the integration_id specified is one that the org in question has access to. That's maybe something that should move to a permissions class though? or since `ExternalIssue` has org/integration id, maybe it's not relevant for `delete`

Reply:
But you could do like:

```python
Integration.objects.get(organizations__organization_id=x)
```

Or however you query the relationship through the through table in Django. It usually takes me a few tries to get it right. It's not a big deal, I just don't personally like it when querying a different model, just to get information on a foreign key, when you can just do the right thing up front.

Reply:
oh yeah that's a better idea. thx!